posthog/temporal/common/clickhouse.py

```diff
@@ -309,6 +309,11 @@ async def apost_query(
             params["query_id"] = query_id
 
         query = self.prepare_query(query, query_parameters)
+        if query_parameters is not None:
```

Comment:
slightly confused here, as `prepare_query` is already formatting the query string but then we're also sending the query parameters as `params`?

Reply:
`prepare_query` is not formatting the queries as the queries have escaped the braces `{{ ... }}`. `prepare_query` was formatting the old queries, which didn't escape the braces. Moreover, `prepare_query` can't (as it stands) format these queries as it cannot resolve the types that clickhouse knows how to resolve: For example, we cannot format the string `"{include_events:Array(String)}"` as that's a clickhouse type, not a Python type.

Reply:
Since we are still formatting the `persons_*` queries we need to keep that formatting here.

Reply:
thanks, that makes sense. Perhaps worth adding a comment for future reference?

Reply:
I've updated the docstring and clarified with a TODO we should let clickhouse handle all parameters.